src/workerd/api/web-socket.c++

```diff
@@ -259,6 +259,10 @@ jsg::Ref<WebSocket> WebSocket::constructor(
     headers.unset(kj::HttpHeaderId::SEC_WEBSOCKET_EXTENSIONS);
   }
 
+  if (!FeatureFlags::get(js).getIncreaseWebsocketMessageSize()) {
+    ws->maxMessageSize = 128u << 20;
+  }
```

Comment:
Nit: Given that the flag is marked experimental, this check feels a bit backwards. The `size_t maxMessageSize` default value should probably be `128u <<20` and this check should be inverted to set `1u << 20` when the flag is set.

Reply:
Shouldn't the `if` condition be reversed? Currently it's `if (!flag)` but it should be `if (flag)`? Then the rest of the logic is correct.

I'd prefer that `1u << 20` is the default, and the flag changes it.